src/driver-steelseries.c

```diff
@@ -878,6 +895,33 @@ steelseries_write_save(struct ratbag_device *device)
 	return 0;
 }
 
+static int
+steelseries_write_resolutions(struct ratbag_profile *profile)
```

Comment:
`steelseries_write_resolutions_v4`

Reply:
if the protocol is now called "dpi" wouldn't `steelseries_write_dpi_v4` be better?

Reply:
Well, you are effectively writing `struct ratbag_resolution` so I would keep the name. I just asked to rename to `STEELSERIES_ID_DPI_PROTOCOL4` to keep the consistency with the other protocol versions.